src/main/java/com/klarna/hiverunner/sql/StatementsSplitter.java

```diff
@@ -50,35 +50,35 @@
                 // Close statement and start a new one
                 case ";":
                     // Only add statement that is not empty
-                    if (isValidStatement(statement)) {
-                        statements.add(statement.trim());
+                    if (isValidStatement(statement.toString())) {
```

Comment:
toString() in StringBuilder creates a copy each time its called so lines 53 and 54 will create two objects. An idea could be to assign the results of calling toString to a local variable instead but I guess we're not too worried about performance and this is more about what is more readable. Either way is quite similar unless anyone else has thoughts?